websites/api-docs-smoke-test/src/content/endpoints/endpoints-for-resource.mdx

```diff
@@ -5,26 +5,26 @@ description: Test endpoints
 
 import { ApiEndpointsForResource } from "/shortcodes"
 
-# /{projectKey}/resource
+# /\{projectKey}/resource
```

Comment:
[EDITORS ALERT] This syntax change needs to be notified to the editors

Reply:
That's a bit pesky but we have no chance.   The curly brace notation for project key and region are all over the place in the docs.